### PR TITLE
Fako/filter sync seeds

### PR DESCRIPTION
**Pull request by fako**

### CODE REVIEW

File: harvester/core/management/commands/harvest_metadata.py [hunk 1]
```diff
@@ -48,7 +48,8 @@ def preprocess_seeds(self, harvest_queryset):
             set_specification = harvest.source.spec
             upserts = []
             deletes = []
-            for seed in get_harvest_seeds(set_specification, harvest.latest_update_at, include_no_url=True):
+            for seed in get_harvest_seeds(harvest.source.repository, harvest.source.spec, harvest.latest_update_at,
```
Comment (naftalivanderloon) [inline harvester/core/management/commands/harvest_metadata.py:51]: naming much clearer!

File: harvester/edurep/tests/test_get_harvest_seeds.py [hunk 2]
```diff
@@ -44,65 +45,69 @@ def check_seed_integrity(self, seeds, include_deleted=True):
...
-        seeds = get_harvest_seeds("surfsharekit", make_aware(datetime(year=1970, month=1, day=1)))
+        seeds = get_harvest_seeds(Repositories.EDUREP, "surfsharekit", make_aware(datetime(year=1970, month=1, day=1)))
         self.assertEqual(len(seeds), 17)
         self.check_seed_integrity(seeds)
 
     def test_get_partial_set(self):
-        seeds = get_harvest_seeds("surfsharekit", make_aware(datetime(year=2020, month=2, day=10, hour=22, minute=22)))
+        seeds = get_harvest_seeds(
+            Repositories.EDUREP,
+            "surfsharekit",
```
Comment (naftalivanderloon) [inline harvester/edurep/tests/test_get_harvest_seeds.py]: maybe create a static string for this one too?  SURFSHAREKIT

File: harvester/sharekit/tasks.py [hunk 3]
```diff
@@ -59,7 +59,12 @@ def sync_sharekit_metadata():
         if len(err) or not len(scc):
             continue
         # Now parse the metadata and update current Collection for this Harvest
-        seeds = SharekitMetadataHarvest.objects.extract_seeds(set_specification, harvest.latest_update_at)
+        seeds = get_harvest_seeds(
+            Repositories.SHAREKIT,
+            set_specification,
```
Comment (naftalivanderloon) [inline harvester/sharekit/tasks.py:64]: not sure what set_specification is here, could this naming be clearer?

File: harvester/sharekit/tests/test_get_harvest_seeds.py [hunk 4]
```diff
@@ -44,35 +45,39 @@ def check_seed_integrity(self, seeds, include_deleted=True):
             self.assertEqual(seed_types["deleted"]["state"], "deleted")
 
     def test_get_complete_set(self):
-        seeds = get_harvest_seeds("edusources", make_aware(datetime(year=1970, month=1, day=1)))
+        seeds = get_harvest_seeds(Repositories.SHAREKIT, "edusources", make_aware(datetime(year=1970, month=1, day=1)))
```
Comment (naftalivanderloon) [inline harvester/sharekit/tests/test_get_harvest_seeds.py]: could we use a static string for edusources?

**Comment by fako** [comment]
@naftalivanderloon documentation about OAI-PMH and "set_specification": http://www.openarchives.org/OAI/openarchivesprotocol.html#Set

**Pull request merged**